packages/theming/atlas/src/themesource/atlas_core/native/variables.ts

```diff
@@ -515,7 +515,8 @@ let floatingActionButton: VariablesFloatingActionButton = {
         fontSize: font.sizeSmall
     },
     secondaryButtonCaptionContainer: {
-        backgroundColor: background.primary
+        backgroundColor: background.primary,
+        borderColor: custom.darkMode ? background.primary : "#eee"
```

Comment:
I think it makes more sense to just use `background.primary` for the border as well, regardless of the theme. So if someone changes `background.primary` they won't run into the same problem because the styles will automatically adapt.

Reply:
I changed to use `background.primary` 👍